component/redis.jsonnet

```diff
@@ -36,9 +36,9 @@ local redisDeployment = kube.Deployment('redis') {
       spec+: {
         containers_+: {
           redis: kube.Container('redis') {
-            image: params.images.redis.image + ':' + params.images.redis.tag,
+            image: '%(registry)s/%(repository)s:%(tag)s' % params.images.redis,
             command: [
-              'redis-server',
+              'redis-server --appendonly yes --maxmemory $$(( $$( cat /sys/fs/cgroup/memory.max ) - 100000000)) --maxmemory-policy allkeys-lru',
```

Comment:
Does that look like a sane command?

Reply:
What happens if there are no limits?

Personally i'd do everything statically in jsonnet where we have an earlier failure path.